buildpipeline/DotNet-CoreFx-Trusted-Windows.json

```diff
@@ -466,6 +466,28 @@
     "PB_CreateHelixArguments": {
       "value": "/t:CloudBuild /p:ArchGroup=x64 /p:ConfigurationGroup=Release /p:\"EnableCloudTest=true\" /p:\"BuildMoniker=none\" /p:\"TargetQueue=Windows.10.Amd64\" /p:\"TestProduct=corefx\" /p:\"TimeoutInSeconds=1200\" /p:\"TargetOS=Windows_NT\" /p:FilterToOSGroup=Windows_NT",
       "allowOverride": true
+    },
+    "PB_OptionalToolSource": {
+      "value": null,
+      "allowOverride": true,
+      "isSecret": true
```

Comment:
Does this mean that all these extra parameters have to be passed from the root pipebuild? If so I'm not sure how much I like that approach. We are definitely having a similar conversation about other tools that you can ask about offline.